### ISSUE TITLE
Missing asterisk-calendar modules

### ISSUE THREAD

**Issue by wardmundy** [IssabelFoundation/issabel#20]
Any chance of adding the Asterisk calendar modules: asterisk-calendar.rpm

There are four calendaring modules that ship with Asterisk that provide support for the following calendaring servers:

iCalendar | res_calendar_icalendar.so
CalDAV    | res_calender_caldav.so
Microsoft Exchange Server            | res_calendar_exchange.so
Microsoft ExchangeWeb Services | res_calendar_ews.so

**Comment by asternic** [comment]
Yes, we will include missing modules on the latest asterisk 11 release, and we will make sure they are added on the version 13 rpm we are working on. Thanks for pointing it out.

**Comment by wardmundy** [comment]
Hi,
Thanks for your response. Just an FYI that we have FreePBX 13 working
without all the nasty stuff and without module signature checking. Thought
it might give you a head start as you move forward.

This week's Nerd Vittles article documents how to grab the installer. Can't
wait for your next release!!

-- Ward Mundy

Ward Mundy & Associates LLC

Follow Us
<https://www.facebook.com/WardMundy>
<https://plus.google.com/u/0/+WardMundy/posts> <https://twitter.com/NerdUno>
<http://youtube.com/wardmundy/>
